data-access-layer/bosh/manifest/Addons.js

```diff
@@ -24,6 +25,15 @@ class Addons {
     }
   }
 
+  shouldEnableConnectivity() {
+    const connectivityAllowedServicesList = _.get(this.context, 'connectivityAllowedServicesList');
```

Comment:
Is this necessary ? If this is for test, then can we not emulate with the params rather ?

Reply:
My understanding was that we need to do this for only postgresql, hence this extra check. WDYT from that perspective?

Reply:
yeah...makes total sense not to open up for all. 🥇